docs/snapshots.asciidoc

```diff
@@ -147,9 +148,18 @@ PUT /_snapshot/my_gcs_repository/test-snapshot
 
 [float]
 [id="{p}-setup-cronjob"]
+==== Periodic snapshots with Snapshot Lifecycle Management
+
+added[7.4.0]
+
+You can use the https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[Snapshot Lifecycle Management APIs] to manage policies for the time and frequency of automatic snapshots.
```

Comment:
```suggestion
You can use the https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-lifecycle-management-api.html[snapshot lifecycle management APIs] to manage policies for the time and frequency of automatic snapshots.
```

Not sure whether we are supposed to use title case for features. It looks a bit odd to me.